provider/cluster/kube/client.go

```diff
@@ -51,10 +50,8 @@ type client struct {
 
 // NewClient returns new Kubernetes Client instance with provided logger, host and ns. Returns error incase of failure
 func NewClient(log log.Logger, host, ns string) (Client, error) {
-	var settings settings
-	if err := env.Parse(&settings); err != nil {
-		return nil, err
-	}
+	settings := newDefaultSettings()
```

Comment:
These need to be able to be configured.  It used to happen above in `env.Parse`, it's not anymore.